tests/profile/serializers/test_indicator_data_serializer.py

```diff
@@ -45,6 +47,48 @@ def metadata(indicator_data):
     return MetaDataFactory(source="A source", url="http://example.com", description="A description", dataset=dataset)
 
 
+
+@pytest.fixture
+def qualitative_dataset(profile):
+    return DatasetFactory(profile=profile, content_type="qualitative")
+
+@pytest.fixture
+def qualitative_groups(qualitative_dataset):
+    return [
+        GroupFactory(
+            dataset=qualitative_dataset,
+            name="content",
+            subindicators=["This is example text", "www.test.com"],
+            can_aggregate=True, can_filter=True),
```

Comment:
does it make sure to aggregate qualitative data?

Reply:
Aggregation is done on frontend @adieyal 
On backend we just send key set as True or False and that can be changed via admin panel
Should we set it to False for qualitative dataset ?